features/base/exec.config

```
@@ -24,6 +24,6 @@ apt-mark manual \
   libfile-find-rule-perl \
   libgdbm-compat4 \
   libnumber-compare-perl \
-  libperl5.34 \
+  libperl5.36 \
```

This should also work:
```suggestion
  "libperl5.*" \
```

---

thanks for the useful hint!